packages/api-client-core/src/GadgetFunctions.ts

```diff
@@ -98,7 +103,7 @@ interface BulkActionWithInputs<OptionsT, VariablesT> {
 export interface ActionFunctionMetadata<OptionsT, VariablesT, SelectionT, SchemaT, DefaultsT, IsBulk> {
   type: "action";
   operationName: string;
-  namespace: string | null;
+  namespace: string | string[] | null;
```

Comment:
This is why we have to use this annoying type -- because existing generated clients with api-client-cores expect the namespace to be a string in the past, we can't just always pass a string array.